services/121-service/src/tokens/africastalking.ts

```diff
@@ -0,0 +1,6 @@
+export const AFRICASTALKING = {
```

Comment:
This doesn't seem like a solution to me.
By committing these files, we'll all end up with "local changes" to these files that we shouldn't commit.
And run the risk of "_committing secrets by accident_" on **every** commit.

Also, the current set-up of the Test-VM (and all the other environments) is to discard any local change (via `git reset --hard`) in the `deploy.sh`-script.

The **real** values of these variables should be ignored by git in any way;
By ignoring specific files(`africastalking.ts` in `.gitignore`) or folders(`tokens/` in `.gitignore`), etc

A downside of that is, on initial check-out/setup, that the code is 'missing' these files, so a manual action is required to create them or copy-them-from-example-file/documentation.
Moving all these things into a single place (`.env`/`.env.example`) makes it more convenient to check for issues and to update-when-your-missing-something. (That was the idea, I thought.)